docs/src/main/sphinx/release/release-442.md

```diff
@@ -0,0 +1,14 @@
+# Release 442 (14 Mar 2024)
+
+## Delta Lake connector
+
+* Fix query failure when a partition value contains forward slash characters. ({issue}`21030`)
+
+## Iceberg connector
+
+* Fix `NullPointerException` query failure. ({issue}`21074`)
```

Comment:
Do we need to add info when that happens?

Reply:
It's during split enumeration, which I don't know how to frame in a user-friendly way?

Reply:
maybe jsut say 

Prevent NullPointerException from split enumeration during query processing

Reply:
Its technical but at least its not vague